src/UI/Series/Details/InfoViewTemplate.hbs

```diff
@@ -55,6 +55,10 @@
             {{#if_eq seasonNumber compare="-1"}}
                 <span class="label label-default">{{title}}</span>
             {{/if_eq}}
+
+            {{#if_eq sceneSeasonNumber compare="-1"}}
```

Comment:
That doesn't change anything with this though, a single record won't ever have both values. If both XEM and our list have the same mapping then we'll get duplicates, but that would happen right now and we'd remove one of them (a big reason why we don't add anime to our list at all).